src/utils/graphql/queries/getApplicationFilterCount.query.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  query getFilteredApplicationCount($filter: ApplicationListShapeFilter, $userId: Int! = 0) {
+    applicationList(filter: $filter, userid: $userId) {
+      totalCount
+    }
```

Comment:
New query just to get counts. Much more efficient than downloading all the data just to use the count values.